pkg/sensors/program/loader.go

```diff
@@ -121,28 +121,73 @@ func RawTracepointAttach(load *Program) AttachFunc {
 	}
 }
 
-func KprobeAttach(load *Program) AttachFunc {
+func KprobeOpen(load *Program) OpenFunc {
+	return func(coll *ebpf.CollectionSpec) error {
+		// Disable loading of override program if it's not needed
```

Comment:
Out of curiosity, why do we need to disable the program?

Reply:
The generic_kprobe_override program is part of bpf_generic_kprobe.o object,
so we want to disable it if the override is not configured. Otherwise it gets
loaded and bpftool will show it. I'll add this comment.